CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -1880,11 +1880,11 @@ int write_headers()
 			return header_size;
 		}
 
-		int flake_encode_init()
+		int Flake_encode_init()

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.